lib/snapshot/utils.go

```diff
@@ -319,3 +320,75 @@ func walkLinks(path, root string, linksWalked *int) (string, error) {
 		return filepath.Join(newdir, newpath), nil
 	}
 }
+
+// compare tar files of memFSNodes.
+func compareNode(node1, node2 *memFSNode, missing1, missing2 map[string]*memFSNode, diff1, diff2 map[string]*tar.Header, path string) {
```

Comment:
doesn't seem like you need "path"

Reply:
I'd like to have it to point the current location in the filesystem although it is same with header's Name.